src/Sentry.Unity/IScopeObserver.cs

```diff
@@ -0,0 +1,43 @@
+namespace Sentry.Unity
+{
+    /// <summary>
+    /// Observer for the sync. of Scopes across SDKs.
+    /// </summary>
+    public interface IScopeObserver
+    {
+        /// <summary>
+        /// Adds a breadcrumb.
+        /// </summary>
+        void AddBreadcrumb(Breadcrumb breadcrumb);
+
+        /// <summary>
+        /// Sets an extra.
+        /// </summary>
+        void SetExtra(string key, string value);
```

Comment:
This interface is defined in the Sentry SDK (added as a submodule) are you redefining it here because of `string` value on Extra?

Reply:
Perhaps best we just change the one in the .NET SDK to avoid having these two interface 'look-alikes' live in the two assemblies

Reply:
reworked as discussed yesterday.